app/invocation/invocation.css

```
@@ -360,7 +360,7 @@
 
   display: grid;
   grid-template-columns: min-content auto;
-  grid-gap: 0 8px;
+  grid-gap: 0 18px;
```

uber nit: we usually keep things in multiples of 8px -- would this look OK at 16px?

```suggestion
  grid-gap: 0 16px;
```

---

Oh yeah totally